docs/Build/grestsvcs.json

```diff
@@ -413,6 +412,49 @@
           }
         }
       }
+    },
+    "/rpc/rewards_history": {
```

Comment:
Did you mean to include changes to `grestsvcs.json` in this PR?

Reply:
ye - can resolve merge conflicts if any on individual PRs, but would be good to get them on frontend soon